Source/DafnyCore/AST/SubstitutingCloner.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+
+namespace Microsoft.Dafny;
+
+class SubstitutingCloner : Cloner {
```

Comment:
Where is this file used? I can't find any mention of it in old code.

Reply:
Removed in https://github.com/dafny-lang/dafny/pull/2734